portality/models/v2/journal.py

```diff
@@ -70,6 +70,22 @@ def find_by_issn(cls, issns, in_doaj=None, max=10):
         records = [cls(**r.get("_source")) for r in result.get("hits", {}).get("hits", [])]
         return records
 
+    @classmethod
+    def find_by_issn_exact(cls, issns, in_doaj=None, max=2):
+        """
+        Finds journal that matches given issns exactly - if no data problems should always be only 1
+        """
+        if not isinstance(issns, list):
+            issns = [issns]
+        if len(issns) > 2:
+            return []
+        q = JournalQuery()
+        q.find_by_issn_exact(issns, in_doaj=in_doaj, max=max)
+        result = cls.query(q=q.query)
+        # create an array of objects, using cls rather than Journal, which means subclasses can use it too
+        records = [cls(**r.get("_source")) for r in result.get("hits", {}).get("hits", [])]
+        return records
```

Comment:
If this is rationalised with the `is_acceptable` function, this method might not be necessary.  there is a `get_journal` method on the article which probably does everything we want, although it only returns a single journal, so we may need to change how we use it, or change that method to give us what we need for the validation.

Reply:
As a summary of our call: we decided to move the ISSN validation against the journals to a separate method, allow this method to return the said journal and leave get_journal unchanged, not to break the current functionality. This can be changed after the clean-up.